Makefile

```diff
@@ -1,6 +1,8 @@
 export PATH := install/bin:$(PATH)
 
+# path to BB runtime's source repo
```

Comment:
I suggest capitalizing this sentence for consistency with the other comment below.